app/controllers/admin/ticket_scannings_controller.rb

```diff
@@ -1,21 +1,16 @@
 module Admin
   class TicketScanningsController < Admin::BaseController
     before_action :authenticate_user!
-    before_action :find_physical_ticket_by_token
-    load_and_authorize_resource through: :physical_ticket, singleton: true, shallow: true
+    load_resource :physical_ticket, find_by: :token
+    # We authorize manually in these actions
+    skip_authorize_resource only: [:create]
```

Comment:
@Ana06 
In line 4 we are loading physical_ticket resource that is needed in our create action so I don't think we can remove that line. line 6 is necessary because of this: https://github.com/openSUSE/osem/blob/master/app/controllers/application_controller.rb#L9
We need to have authorization in every controller and if not we must skip authorization. Here we are manually authorizing therefore we are skipping authorization before method.